RELEASE_PROCESS.md

```diff
@@ -244,3 +244,27 @@ branches were created on demand.
 This model has now largely been replaced by the model we see in
 this document. However you might still encounter leftovers form the old model
 and you are encouraged to create issues regarding inconsistencies.
+
+## Follow up actions
```

Comment:
Should we link to the corresponding step that needs to succeed here or even implicitly put that as a comment on the action run?